cabal-install/tests/UnitTests/Distribution/Client/UserConfig.hs

```diff
@@ -80,6 +83,19 @@ doubleUpdateConfig = bracketTest . const $ do
         listUnique (map show . fromNubList . installSummaryFile $ savedInstallFlags updated)
 
 
+newDefaultConfig :: Assertion
+newDefaultConfig = do
+    sysTmpDir <- getTemporaryDirectory
+    withTempDirectory silent sysTmpDir "cabal-test" $ \tmpDir -> do
+        let configFile  = tmpDir </> "tmp.config"
+        createDirectoryIfMissing True tmpDir
```

Comment:
Yes, that's true. I think I used to create another directory inside 'tmpDir' before writing the config file to that directory. I thought that doing that was unnecessary so I removed it and it seems like I forgot to remove createDirectoryIfMissing. I can fix it tonight if you want.
If you want to just use what's here then I could fix it in a separate commit and do a fork request on that. Is that ok?